src/components/App.js

```diff
@@ -1,15 +1,44 @@
+import React, { Component } from 'react';
 import '../styles/App.css';
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
-// import calculate from '../logic/calculate';
```

Comment:
Kindly remove commented code.

It’s better to delete commented code. If you need to review a previous version of your code, use git history. 

![image](https://user-images.githubusercontent.com/31897434/77977676-3d0dec80-72f8-11ea-8176-dc41cb8135b6.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643